src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs

```diff
@@ -318,14 +318,39 @@ public async Task<string> CreateAnswerAsync()
         /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
         /// <seealso cref="CreateOfferAsync()"/>
         /// <seealso cref="CreateAnswerAsync()"/>
+        /// <seealso cref="GetLocalDescription()"/>
         /// <since_tizen> 9 </since_tizen>
         public void SetLocalDescription(string description)
         {
             ValidateWebRTCState(WebRTCState.Negotiating, WebRTCState.Playing);
 
             ValidationUtil.ValidateIsNullOrEmpty(description, nameof(description));
 
-            NativeWebRTC.SetLocalDescription(Handle, description).ThrowIfFailed("Failed to set description.");
+            NativeWebRTC.SetLocalDescription(Handle, description).ThrowIfFailed("Failed to set local description.");
+        }
+
+        /// <summary>
+        /// Gets the session description for a local peer.
+        /// </summary>
+        /// <returns>The local session descriptino string</returns>
```

Comment:
the typo "descriptino" is still here, same in line 383

Reply:
I'll fix it.